src/components/RoundSelection/RoundSelection.js

```diff
@@ -1,10 +1,65 @@
-import React from 'react';
+import React, { useContext, useEffect } from 'react';
 import './RoundSelection.css';
 import { Option } from '../Option/Option';
+import { AccessTokenContext } from '../../hooks/TokenContext';
+
+const baseURL = process.env.REACT_APP_SPOTIFY_BASE_URL;
 
 const RoundSelection = (props) => {
-    const {setRoute, setRounds} = props;
+    const {setRoute, setRounds, setAlbums} = props;
+    const accessToken = useContext(AccessTokenContext);
     
+    useEffect(() => {
+        const fetchRandomAlbums = async(accessToken) => {
+
+        const queryParam = 'one%'; // to be randomized
+        const offsetParam = 100; // a number between 1 and 2000
+        const typeParam = 'album';
+        const limitParam = 5; // initial batch
+
+        const queryParams = new URLSearchParams({
+            q: queryParam,
+            type: typeParam,
+            limit: limitParam,
+            offset: offsetParam
+          });
+
+        const stringifiedQueryParams = queryParams.toString();
+
+        // append params to baseURL
+        const searchEndpoint = `${baseURL}/search?` + stringifiedQueryParams;
+
+        const albums = await fetch(searchEndpoint, {
+            headers: {
+            'Accept': 'application/json',
+            'Content-Type': 'application/json',
+            'Authorization': 'Bearer ' + accessToken.token
+            }
+        })
+        .then(response => response.json())
+        .then(data => {
```

Comment:
oftentimes when these types of functions get too large or start to be used in multiple places it's good to pull them out into its own file: `createAlbums(data);`. This can then be shared with other APIs that also need to parse albums.

Reply:
Yes, I'm planning to do this since `RoundSelection` is starting to look a bit ugly! I'll clean it up after I'm done with the functionality 👌